### PR TITLE
Add competitor landing page Gurobi

### PR DESCRIPTION
**Pull request by ge0ffrey**

### CODE REVIEW

File: optaplanner-website-root/templates/competitorBase.ftl [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            </div>
+            <div class="col-md-3">
+                <@macros.tryItButton/>
+                <@macros.getStartedButton/>
+                <@macros.latestReleases/>
+                <@macros.servicesOffer/>
+                <@macros.latestEvents/>
+                <@macros.latestBlogPosts/>
+                <@macros.latestVideos/>
+            </div>
```
Comment (ge0ffrey) [inline optaplanner-website-root/templates/competitorBase.ftl:26]: The competitor layouting needs to be cleaned up, when use cases competitor layout is cleaned up. That's out of scope for now.

File: optaplanner-website-root/templates/index.ftl [hunk 2]
```diff
@@ -12,7 +12,7 @@
         <div class="row">
             <div class="col-md-7">
                 <h1>Solve planning and scheduling problems with OptaPlanner</h1>
-                <p><i>An easy-to-use constraint solver with AI algorithms</i></p>
+                <p class="lead">An easy-to-use, open source constraint solver with AI algorithms</p>
```
Comment (ge0ffrey) [inline optaplanner-website-root/templates/index.ftl:15]: Having the word "open source" on the front page, above the fold, turns out to be critical :)

**Review by triceo — COMMENTED** [review]
Looks good, other than the unsubstantiated claim.

File: optaplanner-website-root/content/competitor/gurobi.adoc [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+
+A dedicated link:../community/team.html[core team] of optimization experts
+(employed by https://www.redhat.com[Red Hat]) works full-time on OptaPlanner.
+Also hundreds of external developers have contributed to OptaPlanner.
+Regardless of who wrote the code, an(other) OptaPlanner core developer has reviewed it
+before it is merged into the main repository.
+
+
+[#fasterAndMoreScalable]
+== OptaPlanner is faster and more scalable
```
[the remark this answers is not in the document]
Comment (ge0ffrey) [reply optaplanner-website-root/content/competitor/gurobi.adoc:48]: Yea, we 've seen the proof several times, but we need open benchmarks that users can run themselves to verify on their own. I 'll tone it down until there's open proof.

**Pull request merged**